docs/using/viewing-the-pull-requests-for-a-repository.md

```diff
@@ -0,0 +1,16 @@
+# Viewing the pull requests for a repository
+
+GitHub for Visual Studio exposes the pull requests for the current repository and lets you create new pull requests and review pull requests from other contributors.
+
+1. [Sign in](../getting-started/authenticating-to-github.md) to GitHub.
+2. Open a solution in a GitHub repository.
+3. Open **Team Explorer** and click the **Pull Requests** button to open the **GitHub** pane.
+![Pull Requests button in the Team Explorer pane](images/pull-requests-button2.png)
+4. The open pull requests will be displayed.
+![Pull requests in the GitHub pane](images/pull-request-list-view.png)
+5. Change the pull requests listed by clicking the **Open** link and selecting the filter you want to use from the dropdown with the options *Open/Closed/All*.
+![Pull requests status dropdown filter](images/pull-request-list-filter.png)
+6. Filter pull requests by assignable user by clicking the person symbol and selecting the user you want to view from the dropdown. You can also search for users from the dropdown.
+![Pull requests assignable user dropdown filter](images/pull-request-assignable-user.png)
+7. Click on a pull request title to [view the pull request details and review the pull request](reviewing-a-pull-request-in-visual-studio.md)
```

Comment:
You actually need to double-click (anywhere on the PR item) now, rather than click on the title.